### DISCUSSION TITLE
PyMuPDF-1.22.2: extractText show unreadible output from specific pdf

### DISCUSSION THREAD

**Discussion by zdenop** [pymupdf/PyMuPDF#2378] [Q&A] [answered]
## Describe the bug (mandatory)
I am not able to get a text from this specific pdf. Pdf is correctly displayed by Adobe Reader (also in Chrome, Firefox)
[AD-3.HANO.pdf](https://github.com/pymupdf/PyMuPDF/files/11347181/AD-3.HANO.pdf)

## To Reproduce (mandatory)
```
import fitz

fname = "AD-3.HANO.pdf"

with fitz.open(fname) as doc:
    for page_id in range(doc.page_count):
        page = doc.load_page(page_id)
        content = page.get_textpage()
        print(f"Page {page_id1}:")
        print(content.extractText())
```

## Expected behavior (optional)
Get text out (as with other pdf files)

## Screenshots (optional)
![image](https://user-images.githubusercontent.com/574156/235073133-1ec93a29-d7e2-4058-8bf1-87e4fbe475be.png)

## Your configuration (mandatory)
 - Windows 10 64bit
 - Python 3.9.13 64bit, Python 3.11.2
 - pymupdf-1.22.2 , installation method: (**wheel**).

**Comment by JorjMcKie** [comment]
This is no bug, but a missing feature of the font - intended or not.
Moving this to "Discussions" in case you have more questions.

**Comment by JorjMcKie** [comment, marked as answer]
The PDF uses non-standard encoding which makes it impossible to extract text - not only for (Py-) MuPDF, but also for Adobe Acrobat, Nitro 5, and other PDF viewers.
Confirm this by selecting some text and paste it in some word processor document.

Please be aware that **_showing_** text and **_extracting_** it are feature that do not necessarily be connected - as is the case here.
So all you can do is OCR-ing.
